crates/services/p2p/Cargo.toml

```diff
@@ -58,6 +58,7 @@ rand = "0.8"
 serde = { version = "1.0", features = ["derive"] }
 sha2 = "0.10"
 tokio = { version = "1.21", features = ["sync"] }
+tokio-stream = "0.1"
```

Comment:
You can use `futures::stream::StreamExt` instead of adding a new dependency

Reply:
good one, thanks